### PR TITLE
Update the python plugin to the latest version available in upstream

### PR DESCRIPTION
**Pull request by ericwill**
Signed-off-by: Eric Williams <ericwill@redhat.com>

### What does this PR do?
Updates the python plugin to the latest version available in upstream.

### What issues does this PR fix or reference?
Fixes CRW-962

#### Release Notes
N/A

#### Docs PR (if applicable)
N/A

### CODE REVIEW

**Comment by ericwill** [comment]
@nickboldt I used the 2.6 image here, not sure if that's what I'm supposed to use.

@svor please take a look and also double check that this fixes https://issues.redhat.com/browse/CRW-1235

File: dependencies/che-plugin-registry/v3/plugins/ms-python/python/2020.7.94776/meta.yaml [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-icon: https://raw.githubusercontent.com/redhat-developer/codeready-workspaces/master/dependencies/che-plugin-registry/resources/images/default.svg?sanitize=true
+icon: https://raw.githubusercontent.com/microsoft/vscode-python/2020.7.94776/icon.png
 repository: https://github.com/Microsoft/vscode-python
 category: Language
-firstPublicationDate: '2019-06-20'
+firstPublicationDate: '2020-11-19'
 spec:
   containers:
-  - image: "registry.redhat.io/codeready-workspaces/plugin-java8-rhel8:2.5"
+  - image: "registry.redhat.io/codeready-workspaces/plugin-java8-rhel8:2.6"
```
Comment (nickboldt) [inline dependencies/che-plugin-registry/v3/plugins/ms-python/python/2020.7.94776/meta.yaml:16]: this is correct... once we have one building and it's updated to include python 3.8.

https://quay.io/repository/crw/plugin-java8-rhel8?tag=latest&tab=tags

... SOON! (see https://issues.redhat.com/browse/CRW-922 for WIP to move to new Jenkins)

File: dependencies/che-plugin-registry/v3/plugins/ms-python/python/2020.7.94776/meta.yaml [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
 name: python
-version: 2019.5.18875
+version: 2020.7.94776
 type: VS Code extension
 displayName: Python
 title: Python extension
 description: Linting, Debugging (multi-threaded, remote), Intellisense, code formatting,
   refactoring, unit tests, snippets, and more.
-icon: https://raw.githubusercontent.com/redhat-developer/codeready-workspaces/master/dependencies/che-plugin-registry/resources/images/default.svg?sanitize=true
+icon: https://raw.githubusercontent.com/microsoft/vscode-python/2020.7.94776/icon.png
```
Comment (nickboldt) [inline dependencies/che-plugin-registry/v3/plugins/ms-python/python/2020.7.94776/meta.yaml:10]: we're still using the no-op invisible plugin icons to hide them.

File: dependencies/che-plugin-registry/v3/plugins/ms-python/python/2020.7.94776/meta.yaml (hunk 1 above)
Comment (nickboldt) [reply dependencies/che-plugin-registry/v3/plugins/ms-python/python/2020.7.94776/meta.yaml:16]: see also https://github.com/redhat-developer/codeready-workspaces-images/pull/5/files

File: dependencies/che-plugin-registry/v3/plugins/ms-python/python/2020.7.94776/meta.yaml (hunk 2 above)
Comment (ericwill) [reply dependencies/che-plugin-registry/v3/plugins/ms-python/python/2020.7.94776/meta.yaml:10]: I'll leave this as is for now, if CRW-1385 falls through then I'll update this PR.

**Review by svor — COMMENTED** [review]
I cannot test it until `registry.redhat.io/codeready-workspaces/plugin-java8-rhel8:2.6` is not published

**Comment by nickboldt** [comment]
Should have https://quay.io/repository/crw/plugin-java8-rhel8?tag=2.6&tab=tags available soon but won't have version on registry.redhat.io until we GA in Feb.

Can you rebase this?

**Comment by ericwill** [comment]
> Should have https://quay.io/repository/crw/plugin-java8-rhel8?tag=2.6&tab=tags available soon but won't have version on registry.redhat.io until we GA in Feb.
> 
> Can you rebase this?

Done

**Review by svor — APPROVED** [review]
tested with quay.io/crw/plugin-java8-rhel8:2.6 and it works as expected
screenshot-codeready-orfo apps ci-ln-q406s9t-f76d1 origin-ci-int-gce dev openshift com-2021 01 11-14_52_48

**Comment by ericwill** [comment]
@nickboldt I guess now we need to decide whether we want to push for python 3.8 in 2.6. PR's are here:
* https://github.com/redhat-developer/codeready-workspaces-deprecated/pull/64
* https://github.com/redhat-developer/codeready-workspaces-images/pull/5

**Pull request merged**